src/api/stats/telos/getOmnidexLpApys.js

```diff
@@ -1,62 +1,85 @@
 const BigNumber = require('bignumber.js');
 const { MultiCall } = require('eth-multicall');
-const { avaxWeb3: web3, multicallAddress } = require('../../../utils/web3');
+const { telosWeb3: web3, multicallAddress } = require('../../../utils/web3');
+import { getFarmWithTradingFeesApy } from '../../../utils/getFarmWithTradingFeesApy';
 
-const MasterChef = require('../../../abis/avax/JoeChef.json');
+const ZenMaster = require('../../../abis/telos/ZenMaster.json');
 const ERC20 = require('../../../abis/ERC20.json');
 const fetchPrice = require('../../../utils/fetchPrice');
-const pools = require('../../../data/avax/joeLpPools.json');
-const { BASE_HPY, AVAX_CHAIN_ID } = require('../../../constants');
-const { getTradingFeeAprSushi } = require('../../../utils/getTradingFeeApr');
-import { getFarmWithTradingFeesApy } from '../../../utils/getFarmWithTradingFeesApy';
-import { JOE_LPF } from '../../../constants';
+const pools = require('../../../data/telos/omnidexLpPools.json');
+const { BASE_HPY, TELOS_CHAIN_ID } = require('../../../constants');
+import { OMNIDEX_LPF } from '../../../constants';
 
-const { joeClient } = require('../../../apollo/client');
 const { compound } = require('../../../utils/compound');
 
-const masterchef = '0xd6a4F121CA35509aF06A0Be99093d08462f53052';
-const oracleId = 'JOE';
+const zenmaster = '0x79f5A8BD0d6a00A41EA62cdA426CEf0115117a61';
+const oracleId = 'CHARM';
 const oracle = 'tokens';
 const DECIMALS = '1e18';
 const secondsPerBlock = 1;
 const secondsPerYear = 31536000;
 
-const liquidityProviderFee = JOE_LPF;
-const beefyPerformanceFee = 0.045;
-const shareAfterBeefyPerformanceFee = 1 - beefyPerformanceFee;
+const liquidityProviderFee = OMNIDEX_LPF;
+const yieldHubPerformanceFee = 0.045;
+const shareAfterYieldHubPerformanceFee = 1 - yieldHubPerformanceFee;
 
-const getJoeApys = async () => {
+const getOmnidexLpApys = async () => {
   let apys = {};
   let apyBreakdowns = {};
 
   const tokenPrice = await fetchPrice({ oracle, id: oracleId });
-  const { rewardPerSecond, totalAllocPoint } = await getMasterChefData();
+
+  console.log("Token Price: ", tokenPrice);
+  const { rewardPerSecond, totalAllocPoint } = await getZenMasterData();
+  console.log("Reward Per Second: ", rewardPerSecond);
+  console.log("Total Alloc Point: ", totalAllocPoint);
+
   const { balances, allocPoints } = await getPoolsData(pools);
 
+  console.log("Balances: ", balances);
+  console.log("Alloc Points: ", allocPoints);
+
   const pairAddresses = pools.map(pool => pool.address);
-  const tradingAprs = await getTradingFeeAprSushi(joeClient, pairAddresses, liquidityProviderFee);
 
   for (let i = 0; i < pools.length; i++) {
+
     const pool = pools[i];
 
+    console.log("Pool: ", pool);
```

Comment:
Do we need these log still? Prefer at a minimum fewer logs so that they're less spammy in prod.

Reply:
These were for debugging, let me take them out, forgot to do that